client/app/domain/general-informations/domain-general-informations.controller.js

```diff
@@ -310,6 +314,33 @@ export default class DomainTabGeneralInformationsCtrl {
       });
   }
 
+  getAllOptionDetails(serviceName) {
+    this.loading.options = true;
+    return this.Domain.getOptions(serviceName)
+      .then(options => this.$q.all(
+        _.map(options, option => this.Domain.getOption(serviceName, option)
+          .then((optionDetail) => {
```

Comment:
You can do this to avoid parameter overriding :
```suggestion
          .then(optionDetail => Object.assign({}, optionDetail, { optionActivated: optionDetail.state === this.DOMAIN.DOMAIN_OPTION_STATUS.ACTIVE}
```

Reply:
Thanks for that. This is done